driver/src/test/java/org/neo4j/driver/internal/messaging/request/HelloMessageTest.java

```diff
@@ -39,10 +40,12 @@ void shouldHaveCorrectMetadata() {
         authToken.put("user", value("Alice"));
         authToken.put("credentials", value("SecretPassword"));
 
-        HelloMessage message = new HelloMessage("MyDriver/1.0.2", authToken, Collections.emptyMap(), false, null);
+        HelloMessage message =
+                new HelloMessage("MyDriver/1.0.2", BoltAgentUtil.VALUE, authToken, Collections.emptyMap(), false, null);
```

Comment:
Test with the `BoltAgent` equals to null or with `BoltAgent` with the complete information can be helpful to avoid future regretions.

Reply:
Done.